protected/humhub/components/Application.php

```diff
@@ -28,6 +28,11 @@ class Application extends \yii\web\Application
      */
     private $_homeUrl = null;
 
+    /**
+     * @var array additional password validation rules
+     */
+    public $passwordAdditionalRules = [];
```

Comment:
Not sure if this is the right place to add this. Please review the documentation for config: http://docs.humhub.org/admin-advanced-configuration.html

Reply:
This has been fixed, please review.